src/main/java/com/linkedin/kafka/clients/auditing/abstractimpl/AbstractAuditor.java

```diff
@@ -144,6 +145,8 @@ public void run() {
             LOG.error("Auditor encounter exception.", e);
           }
         }
+      } catch (Throwable t) {
```

Comment:
Don't catch Throwable catch Exception.  There's almost no conditions which are recoverable when a Throwable is thrown that is not an Exception.